Python/Python.sublime-syntax

```diff
@@ -364,6 +364,12 @@ contexts:
 ###[ COMMENTS ]###############################################################
 
   comments:
+    - match: '(# ?)(TODO|ToDo|Todo|todo)\b(.*)'
+      captures:
+        1: comment
```

Comment:
not scoping the `#` as `punctuation.definition.comment` won't play well with features like joining/wrapping lines.

Reply:
Secondly, just `comment` as the scope name is way too lazy. Please take a look at what scope name the other comment-related rules use.